lesson02/index.js

```diff
@@ -0,0 +1,26 @@
+const insertObj = {
+    anchorUrl: '1.html',
+    anchorText: 'これです',
+    imageUrl: 'bookmark.png',
+    imageAlt: 'ブックマーク'
+};
+
+const createElem= (obj)=> {
+    const parentElement = document.getElementById('js-lists');
+    const listElement = document.createElement('li');
+    const anchorElement = document.createElement('a');
+    const imageElement = document.createElement('img');
```

Comment:
IMO
I thought it would be easier to write the constant names concisely.
This is just my impression of how I would do it.

parentElement → ul
listElement → li 
etc...

Reply:
Thanks for the constant name suggestion.
I'll revise it to be more concise.